setup.py

```diff
@@ -14,7 +14,11 @@
     packages=find_packages(exclude=['ez_setup']),
     url='https://github.com/mozilla/sphinx-js',
     include_package_data=True,
-    install_requires=['Jinja2>2.0,<3.0', 'markupsafe==2.0.1', 'parsimonious>=0.9.0,<0.10.0', 'Sphinx>=3.0.0'],
```

Comment:
markupsafe isn't directly used by sphinx-js--pretty sure it's a secondary dependency--so I removed it.

Reply:
Markupsafe is a secondary dependency. It did not follow semver and became incompatible with Jinja at one point.
https://github.com/pallets/jinja/issues/1585

The recommended path forward has been upgrading rather than pinning, but I approved this pin in order to get the tree back to green with the intention to fast follow on getting Jinja, Sphinx, and Parsimonious updated. Good to remove it here while you're in there updating things

Reply:
Got it. I will re-add it with a comment because we don't know what's going to happen going forward.